0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +62,60 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+If the request included a `Callback-Url` header and `Request-Id` header, the recipient handling the ILP Prepare SHOULD choose to handle the packet asynchronously and return the corresponding ILP Fulfill/Reject in a separate outgoing HTTP request.
```

Comment:
I'd like to use a more overt signal for "async-mode", such as what we discussed [in this comment](https://github.com/interledger/rfcs/pull/567#discussion_r376088183).

My rationale is mainly to allow this RFC to be extended later (after this proposal is committed) with other, non-callback-based styles of async communication (e.g., sender polling w\ batch responses). 

Any objection to including a 3rd `prefer:` header as the normatively required field to indicate sync vs async so  we don't have to assume `sync` in the absence of a `Callback-Url` header?

Reply:
I added the `Prefer: respond-async` header to specify asynchronous mode. I'd just note, sync is still assumed in the absence of a `Prefer` header

Reply:
Sounds great!